### ISSUE TITLE
Test data for access editor

### ISSUE THREAD

**Issue by HGustavs** [HGustavs/LenaSYS#7260]
We need better test data for access editor during install in the sql file.

Especially for testing student group allegiance.

**Comment by a16joare** [comment]
Still active from 2019. 
Did not get time to finish this one.

**Comment by b18jonli** [comment]
@HGustavs For what courses and in what way? The way it is currently most users in the courses "Webbprogrammering" and "Webbutveckling - datorgrafik" have an assigned group. Is the goal to have each user in multiple groups? Or to have a larger variety of groups? (currently the only groups are 1 and 2)

Also, I found that the value of "groups" is different depending on if the group was added through the AccessEd or in the testdata; testdata calls group 1 "1" but if the value were set in accessed.js, it would be "No_1". Visually there is no change, but maybe members of group "1" and "No_1" look like they're in different groups (e.g. when grading a "gruppdugga").
